lib/model.js

```diff
@@ -289,6 +295,14 @@ function conformInclude(include, self) {
         include.as = include.association.as;
       }
     } else {
+      if (! include.model && include.as) {
+        var association = _.find(self.associations, 'as', include.as);
+        if (! association) throw new Error('Include unexpected. Unable to find Association with \'as\': ' + include.as);
+
+        include.association = association;
+        return conformInclude(include, self);
+      }
```

Comment:
Yes, that's correct (see test 'should allow me refer to an association using only its alias in object form in eager loading')